sdk/cosmos/azure-cosmos/azure/cosmos/_cosmos_client_connection.py

```diff
@@ -214,7 +214,6 @@ def __init__( # pylint: disable=too-many-statements
                 logger=kwargs.pop("logger", None),
                 enable_diagnostics_logging=kwargs.pop("enable_diagnostics_logging", False),
                 global_endpoint_manager=self._global_endpoint_manager,
-                diagnostics_handler=kwargs.pop("diagnostics_handler", None),
```

Comment:
Isn't this breaking ?

Reply:
This was/is a beta feature that was expected to have significant changes. It was released on a beta build only. The api review for it hasn't happened yet, and now that we won't use it, it also means an API review won't be necessary.